vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/resource/DocumentV1ApiHandler.java

```diff
@@ -113,7 +114,11 @@
 
     private static final Logger log = Logger.getLogger(DocumentV1ApiHandler.class.getName());
     private static final Parser<Integer> integerParser = Integer::parseInt;
-    private static final Parser<Double> doubleParser = Double::parseDouble;
+    private static final Parser<Double> timeoutParser = value -> {
+        int suffixLength = value.endsWith("ms") ? 2 : value.endsWith("s") ? 1 : 0;
```

Comment:
What about us and ns ?
Don't we have som utility method for this ?

Reply:
It doesn't need to be an integer, do I don't see the need for other units. The search handler recognises `ms`, and treats _everything else_ as `s`. I think this is strictly better.

Reply:
Actually, it recognise `ks` and `µs` as well, which is just obscene. I found the code, which is split out somewhere, but it's in the `search` module.

Reply:
May as well use that code, then.

Reply:
(It was already a dependency of this module.)